go.mod

```diff
@@ -39,4 +39,5 @@ require (
 	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+ 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
```

Comment:
Is this addition belongs to this feature?

Reply:
No, fixed